src/main/scala/util/PSDTestMode.scala

```diff
@@ -0,0 +1,19 @@
+// See LICENSE.SiFive for license details.
+
+package freechips.rocketchip.util
+
+import Chisel._
+import freechips.rocketchip.config._
+
+case object IncludePSDTest extends Field[Boolean](false)
+
+class PSDTestModeIO extends Bundle {
+  val test_mode       = Bool(INPUT)
+  val test_mode_reset = Bool(INPUT)
+  // TODO: Clocks?
```

Comment:
consider using `with Clocked` if that makes sense

Reply:
ah, no the comment means that sometimes you may want to provide bypass clocks as well (e.g. for clock muxes). But often those are injected automatically by DFT tools. So it's more like "future work".